infrastructure/src/main/java/org/corfudb/infrastructure/ServerContext.java

```diff
@@ -125,13 +123,14 @@
     @Getter (AccessLevel.PACKAGE)
     private final String localEndpoint;
 
-    @Getter
-    private static final MetricRegistry metrics = new MetricRegistry();
-
     @Getter
     private final Set<String> dsFilePrefixesForCleanup =
             Sets.newHashSet(PREFIX_PHASE_1, PREFIX_PHASE_2, PREFIX_LAYOUTS);
 
+    @Getter
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651478)